Flutter/lib/SettleScreen.dart

```diff
@@ -1,15 +1,11 @@
-import 'package:flutter/material.dart';
-import 'package:tcard/tcard.dart';
-import 'package:groovin_widgets/groovin_widgets.dart';
-import 'widgets/settle_card.dart';
-import 'ResultScreen.dart';
-import 'Settle.dart';
+import 'import_all.dart';
 
 class SettleScreen extends StatefulWidget {
   final Settle settle;
+  final DarkThemeProvider themeChange;
+
+  const SettleScreen(this.settle, this.themeChange);
```

Comment:
Again, seems best if statically accessed through AppTheme

Reply:
Resolved in commit `app-style 2c993e9`